src/utils/text.ts

```diff
@@ -0,0 +1,2 @@
+export const breakStringWithQuotesBySpaces = (string: string) =>
+  string.match(/("[^"]*?"|[^"\s]+)+(?=\s*|\s*$)/g); // source: https://stackoverflow.com/a/16261693/1265472
```

Comment:
See discussion about this regex issue (and the `Code scanning: Inefficient regular expression` error that is raisued for it) in https://github.com/sparkletown/sparkle/pull/1460#discussion_r640977315